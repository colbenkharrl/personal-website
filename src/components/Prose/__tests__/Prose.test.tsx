import { render, screen } from '@testing-library/react';
import * as stories from '../__stories__/Prose.stories';
import { composeStories } from '@storybook/react';

const { Basic } = composeStories(stories);

it('Prose matches snapshot (regression test)', () => {
  render(<Basic data-testid="Prose" />);
  expect(screen.getByTestId('Prose')).toMatchSnapshot(
    'Full component snapshot.',
  );
});
