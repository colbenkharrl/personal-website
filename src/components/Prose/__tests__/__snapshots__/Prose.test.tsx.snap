// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prose matches snapshot (regression test): Full component snapshot. 1`] = `
<div
  class="prose dark:prose-invert"
  data-testid="Prose"
>
  Text.
</div>
`;
